armhf.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM balenalib/armv7hf-debian-node:10.15-jessie-build
+
+RUN ["cross-build-start"]
+
+RUN apt-get update && apt-get -y install build-essential linux-headers-generic gcc g++ ccache git make
```

Comment:
You should probably add ``install --no-install-recommends -qy``.

Reply:
-q is a quiet flag, we're building this on a CI, so what's the point of the quiet flag

Reply:
I find it quiet annoying to debug when the CI fails and I have 1k lines that don't mean anything. Additionally  to that Travis limits the log size to about 4MB and if we exceed that it gets cut off. Source https://docs.travis-ci.com/user/common-build-problems/#log-length-exceeded

So I my opinion it is good practice to silent useless commands or try to shorten them.
``-q`` hides already some not needed output and in my personal images I cranked it up to ``-qq`` to not flood my logs.